fuzzylboxstats.py

```diff
@@ -228,6 +228,13 @@ def run(self, main_samples=None, sub_samples=None):
                 choice = replchoices[i].strip()
                 if debug: print "  Replacing '{}' with '{}':".format(repr(truncate(deleted)), repr(choice))
                 start = time.time()
+                cursor = self.treemanager.cursor
+                if cursor.node.lookup != "<ws>":
+                    # Insert a leading space in situations like `return(x)`.
+                    choice = " " + choice
+                    if cursor.node.symbol.name[cursor.pos:] == "":
```

Comment:
I think this says "only add a space at the end if we've added a space at the beginning" but shouldn't this be an "OR" not and "AND" situation?

Reply:
You're right. What I wanted to do is to insert a trailing whitespace if we are between two tokens and the next token isn't a whitespace (see new commit).